civicrm.module

```diff
@@ -1428,7 +1428,8 @@ function civicrm_modules_uninstalled($modules) {
  * @param array $cid_parts
  */
 function civicrm_metatag_page_cache_cid_parts_alter(&$cid_parts) {
-  if (!empty($cid_parts['path']) && substr($cid_parts['path'], 0, 7) == 'civicrm') {
+  // Avoid substr() because it will not work with Drupal language prefixes.
+  if (!empty($cid_parts['path']) && strpos($cid_parts['path'], 'civicrm') !== FALSE) {
```

Comment:
would it be more accurate to say `strpos($cid_parts['path'], 'civicrm') === 0`? That gives the same functionality as the strpos check.